src/client/java/net/neoforged/neoforge/client/model/loadingplugin/ModelLoadingPlugin.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.client.model.loadingplugin;
+
+public interface ModelLoadingPlugin {
```

Comment:
We could also fold standalone models into the plugin system. (Possibly just deprecating the current API, or rewriting it as a wrapper on top of a plugin).